ecs-cli/modules/command/image_app.go

```diff
@@ -201,6 +229,91 @@ func pullImage(c *cli.Context, rdwr config.ReadWriter, dockerClient dockerclient
 	return nil
 }
 
+type imageInfo struct {
+	RepositoryName string
+	Tag            string
+	ImageDigest    string
+	PushedAt       string
+	Size           string
+}
+
+func getImages(c *cli.Context, rdwr config.ReadWriter, ecrClient ecrclient.Client) error {
+	registryID := c.String(ecscli.RegistryIdFlag)
+	args := c.Args() // repository names
+
+	totalCount := 0
+	w := tabwriter.NewWriter(os.Stdout, 20, 1, 3, ' ', 0)
```

Comment:
Can you const these values so that it's easier to understand what these represent?